Sources/PerformanceTesting/PerformanceTestCase.swift

```diff
@@ -161,3 +161,16 @@ extension Array where Array == Benchmark {
     }
 }
 
+extension Array {
+
+    /// Initializer that fills the array using a generator function. The function
+    /// is passed the index of the current element as an argument.
+    public init(count: Int, fillingWith elementGenerator: (Int) -> Element) {
```

Comment:
Might just call this `generator`, as `element` is implied in the type.

Reply:
done!